tests.py

```diff
@@ -89,9 +89,6 @@ def test_plone_addon_template(self):
                 self.project + '/src/collective/foo/profiles/default/browserlayer.xml',  # noqa
                 self.project + '/src/collective/foo/profiles/default/collectivefoo_default.txt',  # noqa
                 self.project + '/src/collective/foo/profiles/default/metadata.xml',  # noqa
-                self.project + '/src/collective/foo/profiles/uninstall',  # noqa
```

Comment:
@mauritsvanrees I'm wondering why uninstall has been removed here...need to check...